src/XMakeBuildEngine/UnitTests/Definition/ToolsetConfigurationReader_Tests.cs

```diff
@@ -581,12 +581,12 @@ public void ExtensionPathsTest_Basic1()
 #if XPLAT
```

Comment:
Oh, you had to add the `XPLAT` define in this review. Then I think it should be removed entirely. If I'm reading the code right that shouldn't require any code change--it was just there to wall off the platform-detection methods that don't exist in master yet.